src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -306,6 +306,7 @@ internal static int GetWarningLevel(ErrorCode code)
                 case ErrorCode.WRN_CallerLineNumberPreferredOverCallerMemberName:
                 case ErrorCode.WRN_CallerLineNumberPreferredOverCallerFilePath:
                 case ErrorCode.WRN_DelaySignButNoKey:
+                case ErrorCode.WRN_AttributeIgnoredWhenPublicSigning:
```

Comment:
Could you explain? I'm pretty sure this is required, no matter what the underlying number is. Same with the other comment on `DiagnosticTest.cs` (which is within a test that counts up to 9000)... although the position it's inserted at will have to move, since the cases are arranged by error code ascending number.